astroid/brain/brain_regex.py

```diff
@@ -0,0 +1,92 @@
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/astroid/blob/main/CONTRIBUTORS.txt
+
+from __future__ import annotations
+
+from astroid import context, inference_tip, nodes
+from astroid.brain.helpers import register_module_extender
+from astroid.builder import _extract_single_node, parse
+from astroid.const import PY39_PLUS
+from astroid.manager import AstroidManager
+
+
+def _regex_transform() -> nodes.Module:
+    """The RegexFlag enum exposes all its entries by updating globals().
+
+    We hard-code the flags for now.
+    # pylint: disable-next=line-too-long
+    See https://github.com/mrabarnett/mrab-regex/blob/cc392629b28ed0aeb8bf6d46f826ff9e87aac551/regex_3/regex.py#L198
```

Comment:
```suggestion
    See https://github.com/mrabarnett/mrab-regex/blob/2022.10.31/regex_3/regex.py#L200
```
Tag name are shorter than full sha :)